src/Talis/Persona/Client/Users.php

```diff
@@ -16,16 +16,20 @@ class Users extends Base
     public function getUserByGupid($gupid, $token, $cacheTTL = 300)
     {
         if (!is_string($gupid) || trim($gupid) === '') {
-            $this->getLogger()->error("Invalid gupid $gupid");
             throw new \InvalidArgumentException("Invalid gupid");
         }
         if (!is_string($token) || trim($token) === '') {
-            $this->getLogger()->error("Invalid token $token");
```

Comment:
How come these log statements have been removed? If a exception is thrown on line 114 a logging call is made. We should either include no logging, or log all errors and be consistent with our approach.

Personally I would prefer if we log the errors for debugging purposes.